pandas/tseries/tests/test_timeseries.py

```diff
@@ -949,12 +949,29 @@ def test_nat_vector_field_access(self):
     def test_nat_scalar_field_access(self):
         fields = ['year', 'quarter', 'month', 'day', 'hour',
                   'minute', 'second', 'microsecond', 'nanosecond',
-                  'week', 'dayofyear', 'days_in_month']
+                  'week', 'dayofyear', 'days_in_month', 'daysinmonth',
+                  'dayofweek']
         for field in fields:
             result = getattr(NaT, field)
             self.assertTrue(np.isnan(result))
 
-        self.assertTrue(np.isnan(NaT.weekday()))
+    def test_NaT_methods(self):
+        # GH 9513
+        methods_raise = ['astimezone', 'combine', 'ctime', 'dst', 'fromordinal',
+                         'fromtimestamp', 'isocalendar', 'isoformat',
+                         'isoweekday', 'now', 'replace', 'strftime', 'strptime',
+                         'timestamp', 'timetuple', 'timetz', 'to_datetime',
+                         'to_datetime64', 'today', 'tzname', 'utcfromtimestamp',
```

Comment:
can you also add a test that makes sure these are the same results for a ``Timestamp('NaT')`` and ``func(DatetimeIndex([np.nan]))[0]``